utils/casadm.8

```diff
@@ -172,7 +172,7 @@ command there is a different list of available options:
 
 .TP
 .B -d, --cache-device <DEVICE>
-Path to caching device to be used e.g. SSD device (/dev/sdb).
+Path to caching device to be used e.g. SSD device (/dev/disk/by-id/nvme-INTEL_SSDP...).
```

Comment:
Maybe we should state "by-id" requirement here explicitly: 
Path to caching device using by-id link (e.g. /dev/disk/by-id/nvme-INTEL_SSDP...).

Reply:
thx